string station
string task_id
---
int8 result_id